tests/embedding_net_test.py

```diff
@@ -601,6 +602,9 @@ def test_scan(
 ):
     """Test the scan forward pass of the LRU layer, should be equal to the loop."""
     # causal
+    # Skip test if Python version >= 3.13 since torch.compiler is not supported
+    if sys.version_info >= (3, 13):
+        pytest.xfail("torch.compiler not supported on Python >= 3.13")
```

Comment:
I think it would be better moving this above into something like `pytest.mark.xfail(condition=sys.version_info >= (3, 13)`. do you agree?

Reply:
For this tests it makes sense, but I detected other test that also fail if they use "scan" (as this internally relies on dynamo). I will put it there inside the tests as it depends on the arguments.